.venv/.gitignore

```diff
@@ -1,2 +0,0 @@
-*
-!.gitignore
```

Comment:
Why was this removed, we need the `.venv` dir so that pipenv puts the virtual env locally.

Reply:
It was by mistake, sorry. I restored the .venv/ directory with a git checkout from the main branch